pilot/lib/pilot/build_manager.rb

```diff
@@ -142,10 +143,14 @@ def wait_for_processing_build(options, platform)
         #  have a nil build
         if app.build_trains(platform: platform).count == 0
           UI.message("New application; waiting for build train to appear on iTunes Connect")
-        else
+        elsif !found_build
           builds = app.all_processing_builds(platform: platform)
           break if builds.count == 0
           latest_build = builds.last
+          found_build = true
+        else
```

Comment:
would this alternative code work? (one liner)

* remove the found_build variable
* use
```ruby
    else
      builds = app.all_processing_builds(platform: platform)
      break if builds.count == 0
      latest_build = builds.last		
      next unless builds.include? latest_build # skip broken builds and retry
```